src/cljs/org/broadinstitute/firecloud_ui/testrunner.cljs

```diff
@@ -0,0 +1,10 @@
+(ns org.broadinstitute.firecloud-ui.testrunner
+  (:require
+   cljs.test
+   org.broadinstitute.firecloud-ui.configtest
+   org.broadinstitute.firecloud-ui.main
+   org.broadinstitute.firecloud-ui.testutils
+   ))
+
+(defn run-all-tests []
+  (cljs.test/run-tests 'org.broadinstitute.firecloud-ui.configtest))
```

Comment:
Do all tests have to be manually added here? Is there any way to auto-discover tests?